environment/stanford_sites/webauth.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Webauth login
+  In order to have access to permissions and use other webauth features
+  As an end user
+  I want to be able to login to the site using webauth
+
+  @safe
+  Scenario: Verify that /user page has the appropriate content
+    Given I am on "user"
+    Then I should see the text "WebAuth Login"
+    And I should see the text "Local User Login"
```

Comment:
I created it in the jse product directory first and tested it there. But I see what you mean.  I'm unable to test any files that are in the environments directory.  Even if I symlink it I get a no specifications found at path(s)... error.  Is there a workaround for this?